### ISSUE TITLE
Docker-compose: "Additional property env_file is not allowed"

### ISSUE THREAD

**Issue by iJackWilson** [nccgroup/ScoutSuite#1406]
When attempting to build the ScoutSuite Docker image from source (following step 2 in [the wiki](https://github.com/nccgroup/ScoutSuite/wiki/Docker-Image#running-the-container)). I receive the below error:

`services.ncc-scoutsuite.build Additional property env_file is not allowed`

I believe this is a result of [two extra unexpected lines in the `docker-compose.yml` file](https://github.com/nccgroup/ScoutSuite/blob/master/docker/docker-compose.yaml#L19-L20).

**To Reproduce**

On a clean system (e.g. one that does not cache a previous build of the ScoutSuite image), follow the documentation:
* Clone ScoutSuite repository.
* Change to the `ScoutSuite/docker` folder.
* Run `docker-compose up --build`.
* Receive the error `services.ncc-scoutsuite.build Additional property env_file is not allowed`.

**Additional context**

I was able to successfully build the Docker container by removing [these two lines](https://github.com/nccgroup/ScoutSuite/blob/master/docker/docker-compose.yaml#L19-L20) from the `docker-compose.yaml` file.

<img width="762" alt="Screenshot 2022-03-17 at 11 48 32" src="https://user-images.githubusercontent.com/16834494/158810728-5bb94153-8bab-4270-8c61-2129f8c01ffe.png">

![image](https://user-images.githubusercontent.com/16834494/158810753-004ea35a-735a-424b-96f9-6f18e69d63b4.png)

<img width="1440" alt="Screenshot 2022-03-17 at 11 49 10" src="https://user-images.githubusercontent.com/16834494/158810638-3c195010-9b7f-4376-87a6-8dbcc3844d4a.png">

**Comment by ben-elttam** [comment]
Alternative right now is  build using `docker` command rather than `docker-compose` like this:

```bash
cd ./docker/
docker build -t nccgroup/scoutsuite .
```

**Comment by rossja** [comment]
this appears to be fixed. can you confirm whether this is still occurring?

**Comment by M-Davies** [comment]
Still occurring for me
```
PS C:\Tools\Recon\ScoutSuite\docker> docker-compose up --build
WARNING: The VCS_REF variable is not set. Defaulting to a blank string.
WARNING: The VCS_URL variable is not set. Defaulting to a blank string.
WARNING: The VERSION variable is not set. Defaulting to a blank string.
WARNING: The BUILD_DATE variable is not set. Defaulting to a blank string.
WARNING: The VENDOR variable is not set. Defaulting to a blank string.
WARNING: The NAME variable is not set. Defaulting to a blank string.
WARNING: The IMAGE_NAME variable is not set. Defaulting to a blank string.
WARNING: The DESCRIPTION variable is not set. Defaulting to a blank string.
ERROR: The Compose file '.\docker-compose.yaml' is invalid because:
services.ncc-scoutsuite.build contains unsupported option: 'env_file'
```

**Comment by M-Davies** [comment]
Theoretically, this extra addition isn't needed? It's [already declared higher up in the file](https://github.com/nccgroup/ScoutSuite/blob/4300fc0440db766fafb0db81de7c954534b0349c/docker/docker-compose.yaml#L5-L6) and the [documentation](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option) seems to suggest that's where it needs to be anyways.

**Comment by fernando-gallego** [comment]
Solved in version 5.12.0
